Translations update from Fedora Weblate

weblate opened the pull request:
Translations update from [Fedora Weblate](https://translate.fedoraproject.org) for [gss-ntlmssp/main](https://translate.fedoraproject.org/projects/gss-ntlmssp/master/).



Current translation status:

![Weblate translation status](https://translate.fedoraproject.org/widget/gss-ntlmssp/master/horizontal-auto.svg)

simo5 commented:
I do not know how much I can trust google translate to check these, but it seem that "context" has been translated as "content", which seems incorrect.

neko0xff committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (30 of 30 strings)

Added translation using Weblate (Chinese (Traditional Han script))

Co-authored-by: hsu zangmen <chzang55@gmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/gss-ntlmssp/master/zh_Hant/
Translation: gss-ntlmssp/main

arifpedia committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (30 of 30 strings)

Added translation using Weblate (Indonesian)

Co-authored-by: Arif Budiman <arifpedia@gmail.com>
Translate-URL: https://translate.fedoraproject.org/projects/gss-ntlmssp/master/id/
Translation: gss-ntlmssp/main
